jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepExecution.java

```diff
@@ -668,7 +684,7 @@ private String setupSettingFile(@Nonnull Collection<Credentials> credentials) th
             FilePath settings;
             if ((settings = ws.child(settingsPath)).exists()) {
                 // Settings file residing on the agent
-                console.format("[withMaven] use Maven settings provided by the Jenkins global configuration on the build agent '%s' %n", settingsPath);
+                console.format("[withMaven] use Maven settings on the build agent '%s' %n", settingsPath);

```

Comment:
Ditto.
Shall we indicate if the config has specified "globally" or by a "folder"?